stdlib/contrib/rhajek/bigpanda/bigpanda.flux

```diff
@@ -27,12 +39,16 @@ statusFromLevel = (level) => {
     return sev
 }
 
-// `sendAlert` sends a single alert to BigPanda as described in https://docs.bigpanda.io/reference#alerts API. 
-// `token` - string - BigPanda authorization Bearer token
-// `url` - string - base URL of [BigPanda API](https://docs.bigpanda.io/reference#alerts).
-// `appKey` - string - BigPanda App Key.
-// `status` - string - Status of the BigPanda alert. One of ok, critical, warning, acknowledged.
-// `rec` - record - additional data appended to alert
+// sendAlert sends a single alert to BigPanda as described in the
+// bigpanda [API reference](https://docs.bigpanda.io/reference#alerts API). 
+//
+// ## Parameters
+// 
+// - token: BigPanda authorization Bearer token
+// - url: base URL of [BigPanda API](https://docs.bigpanda.io/reference#alerts).
+// - appKey: BigPanda App Key.
+// - status: Status of the BigPanda alert. One of ok, critical, warning, acknowledged.
+// - rec: Additional data appended to alert.
```

Comment:
- Update with parameter descriptions and examples from https://docs.influxdata.com/flux/v0.x/stdlib/contrib/rhajek/bigpanda/sendalert/
- Add the `tags: single notification` metadata